db/migrate/20240109223238_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.integer :post_id
```

Comment:
👍 Good job on creating the migration file for the `comments` table! However, there's a crucial element missing in the current setup.

- [ ] Upon reviewing the database diagram, it's evident that the `comments` table should have a one-to-many relationship with the `users` table and another one-to-many relationship with the `posts` table. This association is essential for linking comments to their respective posts and users. To address this, please follow the [official documentation](https://guides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) on creating associations using references. Update your migration file accordingly to establish the relationship between the `comments` to `users` and `posts` tables using `user_id` and `post_id`. After making the necessary changes, run the migration files to ensure that the code works successfully.
![image](https://github.com/hamidazim321/blog_app/assets/55840999/158ed35c-3062-45c9-948d-6cd17624e3f0)

Keep up the good work! If you have any questions or need further guidance, feel free to ask. 🚀👍

Reply:
Thanks for the review @shahadat3669 , but as you know, when we put the belongs_to in migration, it's just add  tablename_id as a new column, and the proper association is set in model, so that why I decided to create all the foreign column manually and set the association in the column ... is it bad to do that ??

Reply:
@francksefu  Thank you for your work on setting up associations in the model. However, for this milestone, it's crucial to follow the project requirements of creating and running necessary migration files. This ensures consistency and clarity in the codebase, aligning with specified project instructions.
![image](https://github.com/francksefu/blog_web/assets/55840999/d075e39a-6055-4010-adaa-b4425ee7cc7a)

In Ruby on Rails, having relationships in both the database and model layers is essential:
- **Database Level:** Ensures data integrity and optimizes queries.
- **Model Level:** Enhances code readability and maintainability through object-oriented abstraction.

Since this milestone focuses on migration files, please update your code accordingly to meet the specified requirements. If you have any questions, feel free to ask. Keep up the good work!"